packages/slice-machine/src/modules/screenshots/actions.ts

```diff
@@ -16,7 +16,11 @@ export const generateSliceScreenshotCreator = createAsyncAction(
     component: ComponentUI;
     screenDimensions: ScreenDimensions;
   },
-  { screenshot: ScreenshotUI | null; component: ComponentUI }
+  {
+    screenshot: ScreenshotUI | null;
+    component: ComponentUI;
+    variationId: string;
```

Comment:
how come this change is not merged yet ?

Reply:
The action was previously not exposing the variation ID which we needed in the saga to update the library properly. So this is a new change

Reply:
My bad, I thought this change in another PR, so i was surprised that it owuld be here too.